docs/model/product-feeds.md

```diff
@@ -40,6 +40,10 @@ In such a case, this feed will be generated every four hours.
 Feeds have their default times set in their own `services.yaml` files, but can be easily changed in your project's `feed.yaml` file.
 You only need to copy service definition from feed `services.yaml` file and change the hours and minutes to expected ones.
 
+## How to run feed outside scheduled time?
+
+If you want to run feed outside scheduled time, you can use the `php bin/console shopsys:feed-schedule` command with `--feed-name` argument to schedule one feed or `--all` argument to schedule all feeds and then run `php bin/console shopsys:cron --module="Shopsys\FrameworkBundle\Model\Feed\FeedCronModule" --instance-name=export` to generate the feeds.
```

Comment:
could you please also implement a button for this in administration? It should probably replace the current "generate" button in `admin/feed/list/`

Reply:
The current generate button is still working.

Reply:
ok, I think the current button is not usable for the bigger amount of products, but it is not important right now. We can keep it there. But could we add the button for the new functionality, please? I think it would be appreciated (same as the "schedule" CRON module is at the moment), maybe @pk16011990 can add his opinion here?

Reply:
I would like this button :)

Reply:
Added ability to do that